demisto_sdk/commands/common/constants.py

```diff
@@ -1776,3 +1851,15 @@ class ParameterType(Enum):
 
 NO_TESTS_DEPRECATED = "No tests (deprecated)"
 NATIVE_IMAGE_FILE_NAME = "docker_native_image_config.json"
+
+PACK_ROOT_FILE_NAMES = {
```

Comment:
what do you mean?

Reply:
you added Pipfile, ".gitignore", and Pipfile.lock as strings rather than adding their constants.
let's create a constant for these files and use the constant instead.

Reply:
Nice, done.